app/uk/gov/hmrc/tariffclassificationfrontend/service/EventsService.scala

```diff
@@ -32,4 +34,13 @@ class EventsService @Inject()(connector: BindingTariffClassificationConnector) {
     connector.findEvents(reference)
   }
 
+  def addNote(reference: String, note: String, operator: Operator, clock: Clock = Clock.systemDefaultZone())(implicit hc: HeaderCarrier): Future[Unit] = {
+    val event = NewEventRequest(Note(Some(note)), operator, ZonedDateTime.now(clock))
+    connector.createEvent(reference, event)
+      .recover {
+        case t: Throwable => Logger.error(s"Could not create Event for case [$reference] with payload [$event]", t)
```

Comment:
Is there a specific reason why you added this `recover` block?
If there is an error, I would prefer the future to fail.
We have the errors being logged in Kibana by the default error handler (defined in `application.conf`)

Typically we define `recover` blocks only for custom things ( for example if we want to add extra logging in the scheduled tasks). 
This seems something standard.

Reply:
I was following the pattern from CasesService. I will come see you to discuss another way to code it

Reply:
Fixed